core/src/components/list/test/lines/list.e2e.ts

```diff
@@ -26,3 +26,154 @@ configs().forEach(({ title, screenshot, config }) => {
     });
   });
 });
+
+/**
+ * Padding and border color ensures the bottom border can be easily seen if it regresses.
+ * The background color ensures that any border radius values can be seen.
+ */
+configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('list: lines with children'), () => {
+    test('only item in inset list should not have line', async ({ page }) => {
```

Comment:
Channeling my inner Maria to ask for a test annotation. 🙂

Reply:
sorry I'll add that 😂